lxd/instance/drivers/driver_qemu.go

```diff
@@ -1480,6 +1573,19 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 	}
 
 	cpuType := "host"
+
+	// Get CPU flags if clustered and migration is enabled.
+	if d.state.ServerClustered && util.IsTrue(d.expandedConfig["migration.stateful"]) {
+		cpuFlags, err := d.getClusterCPUFlags()
+		if err != nil {
+			op.Done(err)
+			return err
+		}
+
+		cpuType = "kvm64"
```

Comment:
this is hardcoded in at least 2 places so far, so bearing in mind https://github.com/canonical/lxd/pull/15351/commits/7f22559a7a3354d020969879cb388242a1c892ab#r2034758971 we should at least have a constant for it.